docs/cli.rst

```diff
@@ -1,11 +1,293 @@
-The ``cci`` command line
+The ``cci`` Command Line
 ========================
 
-Basic operation
+
+
+Basic Operation
 ---------------
 
-Getting help
-------------
+.. tip:: If you're new to working with command line interfaces, the `Install Visual Studio Code <https://trailhead.salesforce.com/content/learn/modules/cumulusci-setup/review-base-requirements-install-visual-studio-code?trail_id=build-applications-with-cumulusci>`_ module covers installing and opening a terminal window in Visual Studio Code.
+
+After installing CumulusCI, use the ``cci`` command in your terminal or command prompt to interact with it.
+
+On macOS, access the terminal via ``Terminal.app``. On Windows, open ``cmd.exe``. On Linux, use your preferred terminal application.
+
+To see all available commands, type ``cci`` in your terminal.
+
+.. code-block:: console
+
+    $ cci
+    Usage: cci [OPTIONS] COMMAND [ARGS]...
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    error    Get or share information about an error
+    flow     Commands for finding and running flows for a project
+    org      Commands for connecting and interacting with Salesforce orgs
+    project  Commands for interacting with project repository configurations
+    service  Commands for connecting services to the keychain
+    shell    Drop into a Python shell
+    task     Commands for finding and running tasks for a project
+    version  Print the current version of CumulusCI
+
+To retrieve information on a specific command, type ``cci <command>``.
+
+    Example: ``cci task``
+
+.. code-block:: console
+
+    $ cci task
+    Usage: cci task [OPTIONS] COMMAND [ARGS]...
+
+    Commands for finding and running tasks for a project
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    doc   Exports RST format documentation for all tasks
+    info  Displays information for a task
+    list  List available tasks for the current context
+    run   Runs a task
+
+The ``cci task`` command also has many useful subcommands, such as ``cci task info <name>``.
+
+
+
+Get Help Running Tasks
+----------------------
+
+If you're not certain about what a specific command does, use the ``--help`` flag to get more information. 
+
+    Example: ``cci task run --help``
+
+For top-level commands (that don't do anything on their own but have additional subcommands underneath them), this output will be the same with or without the ``--help`` flag.
+
+.. code-block::
+
+    $ cci task info <name> --help
+     Usage: cci task run [OPTIONS] TASK_NAME
+
+When the ``--help`` flag is specified for a fully realized command (one that is executable) (TODO: what is correct word here?), the output will display:
```

Comment:
Just noting the TODO when we get more eyes on this.

Reply:
The following list seems to be accurate for commands that are groups too; do we need to make the distinction?

Reply:
Agree. We can just say: 
```suggestion
When the ``--help`` flag is specified for a command the output will display:
```